src/Applications/GEOSdas_App/testsuites/CMakeLists.txt

```diff
@@ -33,16 +33,33 @@ if(GIT_FOUND)
       else ()
          message(FATAL_ERROR "This should not be reached")
       endif ()
+   else ()
+      set (GIT_TAG_OR_REV GEOSadas-${CMAKE_PROJECT_VERSION})
    endif ()
+
+   message("Setting fvID to fvsetup SHA1 (1st 10 digits)")
+   execute_process(
+      COMMAND ${GIT_EXECUTABLE} hash-object src/Applications/GEOSdas_App/fvsetup
+      COMMAND cut -c1-10
+      WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
+      OUTPUT_VARIABLE fvID
+      OUTPUT_STRIP_TRAILING_WHITESPACE
+   )
+   message("fvID=${fvID}")
```

Comment:
I am concerned about this block of code.   We generally don't want to touch the source directory,  and in a mepo context this is particularly problematic as it would be at the level of the fixture.   Slightly preferable would be ${CMAKE_CURRENT_SOURCE_DIR}, but really something in the binary tree would be preferred.

Perhaps someone could explain to me why execution in the source directory is desired/essential?